pyk/src/pyk/kcfg/exploration.py

```diff
@@ -109,5 +109,5 @@ def to_dict(self) -> dict[str, Any]:
     #
 
     # Minimizing the KCFG
-    def minimize_kcfg(self) -> None:
-        KCFGMinimizer(self.kcfg).minimize()
+    def minimize_kcfg(self, merge: bool = True) -> None:
+        KCFGMinimizer(self.kcfg).minimize(merge=merge)
```

Comment:
```suggestion
    def minimize_kcfg(self, merge: bool = False) -> None:
        KCFGMinimizer(self.kcfg).minimize(merge=merge)
```

The defaults should probably match to set them both to `False`.